tests/cases/fourslash/extract-method-not-for-token-expression-statement.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path='fourslash.ts' />
```

Comment:
Alternatively, you could use a test like https://github.com/Microsoft/TypeScript/blob/2a70bf51589c4312a9329bce68cb2b5c9876e613/src/harness/unittests/extractMethods.ts#L394.